uint8 sonar1
uint8 sonar2
uint8 sonar3
uint8 sonar4
uint8 sonar5
uint8 sonar6
uint8 sonar7
uint8 sonar8
